src/e2e/iOS/__tests__/caret.ts

```diff
@@ -40,10 +39,8 @@ it('Enter edit mode', async () => {
 })
 
 it('Preserve Editing: true', async () => {
-  await gesture(gestures.newThought)
-  await editThought('foo')
-  await gesture(gestures.newSubThought)
-  await editThought('bar')
+  await newThought('foo')
+  await newThought('bar', true)
```

Comment:
Let's use an Options object `{ insertNewSubthought?: boolean }`. The option nicely mirrors the option to the `newThought` reducers.

In my experience I have found that Options objects are almost always preferable to optional arguments. They are self-documenting at the call-site since the option name is explicitly provided, and they provide an easy way to add additional options without order mattering.

Reply:
Done in fda6386f77

Reply:
Done in `fda6386f77`